tcf_website/templates/reviews/review_form_content.html

```diff
@@ -40,7 +40,7 @@ <h2 class="h4 review-form-header form-group col-12 my-4">
     Tell us more about the course and instructor</h2>
   <div class="form-group col mb-3">
     <textarea class="form-control" id="reviewtext" name="text" rows="5"
-              placeholder="(Aim for 3 sentences or more!)" minlength="100" maxlength="5000">{{ form.text.value }}</textarea>
+              placeholder="(Aim for 300 words or more! Include information about the professor's teaching style and ability to convey information, workload and assignment types. Include what you liked about the course and what could be improved on. Well written reviews help other students get a good understanding of classes they're considering taking.)" minlength="100" maxlength="5000">{{ form.text.value }}</textarea>

```

Comment:
Sorry for the pedantry, but I believe the term is `well-written`, not `well written`. Also, consider adding an exclamation point at the end to sound fRiEnDLy?

Reply:
Made the updates!